testbed/infrastructure.ts

```diff
@@ -41,5 +41,14 @@ export class Infrastructure extends cdk.Construct {
             minSize: 5,
             maxSize: 20
         });
+        
+        //service account used by tekton workflows.
+        const sa = this.cluster.addServiceAccount('test-executor', {
+            name: "test-executor",
+        });
+
+        sa.role.addManagedPolicy({
+            managedPolicyArn: `arn:aws:iam::aws:policy/AdministratorAccess`
```

Comment:
Can we cut an issue to scope this down? We should be much more judicious about assigning permissions to this service account.

Reply:
Tekton tasks should be able to do anything and everything for the given account. Like creating eks clusters, configuring various types of resources like LB, creating ASG's etc etc for KIT clusters etc. List goes on an on, and we need to keep updating this as and when underlying tools change their scope. Given this is purely used for testing and it's a test account, wdyt ?